emission/analysis/section_features.py

```diff
@@ -176,13 +238,43 @@ def calAvgSpeed(segment):
 # segment0: speed0 / (t1 - t0), segment1: (speed1 - speed0)/(t2-t1),
 # segment2: (speed2 - speed1) / (t3-t2)
 
+def calAccels(segment):
+  from dateutil import parser
+  try:
+    ts = esta.TimeSeries.get_time_series(segment['user_id'])
+    entries = list(ts.find_entries(['background/filtered_location'], time_query=None))
+  except:
+    return []
+
+  speeds = calSpeeds(segment)
+ 
+  if speeds is None or len(speeds) == 0:
+    return None
+
+  accel = np.zeros(len(speeds) - 1)
+  prevSpeed = 0
+  for (i, speed) in enumerate(speeds[0:-1]):
+    currSpeed = speed # speed0
+    speedDelta = currSpeed - prevSpeed # (speed0 - 0)
+    accel[i] = speedDelta
+    # t1 - t0
+
+    timeDelta = entries[i+1]['data']['ts'] - entries[i]['data']['ts']
```

Comment:
seems reasonable, but my brain is not working and doesn't seem to be identical to original code.
Are you sure that the acceleration you are computing is in the correct direction?